django/core/cache/backends/memcached.py

```diff
@@ -213,3 +213,14 @@ def close(self, **kwargs):
         # libmemcached manages its own connections. Don't call disconnect_all()
         # as it resets the failover state and creates unnecessary reconnects.
         pass
+
+    def set(self, key, value, timeout=DEFAULT_TIMEOUT, version=None):
+        """
+        PyLibMC throws an exception if the stored value is too large,
+        python-memcache instead returns False. We need to override set
+        here to account for the difference.
+        """
+        try:
+            super().set(key, value, timeout=timeout, version=version)
+        except self._lib.TooBig:
```

Comment:
This breaks support for `pylibmc < 1.5.2` we should document this change in release notes and tests requirements.

Reply:
Thanks @felixxm, 
> we should document this change in release notes

Should we mention the new behavior(Handling the TooBig exception) or mention the `pylibmc` version update?

about the `tests requirements`, Do you mean to change `pylibmc; sys.platform != 'win32'` to `pylibmc >= 1.5.2; sys.platform != 'win32'` in `tests/tequirements/py3`?

Reply:
> Should we mention the new behavior(Handling the TooBig exception) or mention the `pylibmc` version update?

Removing support for older ``pylibmc`` versions, e.g. ``` * Support for ``pylibmc`` < 1.5.2 is removed.```

> about the `tests requirements`, Do you mean to change `pylibmc; sys.platform != 'win32'` to `pylibmc >= 1.5.2; sys.platform != 'win32'` in `tests/tequirements/py3`?

Yes